weeks/week07/week07.md

```diff
@@ -1,3 +1,38 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## General information:
+Playwright 
+It is the powerful tool for Web Test automation (UI and API) in different programming languages:
```

Comment:
I think this line should be right after the word "Playwright" in line 4.

Reply:
Done. Thanks. It is better to be reviewed later upon completion of the page for efforts and time-saving reasons.